app/views/layouts/supplejack_api/_top_nav.html.erb

```diff
@@ -1,10 +1,10 @@
-<%#   
-The majority of the Supplejack API code is Crown copyright (C) 2014, New Zealand Government, 
+<%#
+The majority of the Supplejack API code is Crown copyright (C) 2014, New Zealand Government,
 and is licensed under the GNU General Public License, version 3.
-One component is a third party component. See https://github.com/DigitalNZ/supplejack_api for details. 
+One component is a third party component. See https://github.com/DigitalNZ/supplejack_api for details.
 
-Supplejack was created by DigitalNZ at the National Library of NZ and 
-the Department of Internal Affairs. http://digitalnz.org/supplejack 
+Supplejack was created by DigitalNZ at the National Library of NZ and
```

Comment:
same here, can we delete all the views now?  Is it purely an api or do we have a few more steps to go?